ImguiMapEditor/IO/Otbm/OtbmWriter.cpp

```
@@ -164,13 +323,28 @@ bool writeTile(
     // Tile flags - only save map flags, not editor flags (Selected, Modified)
     // Map flags are bits 0-5 (ProtectionZone, NoPvP, NoLogout, PvpZone, Refresh)
     // Editor flags are bits 8+ (Selected=0x100, Modified=0x200)
-    constexpr uint32_t MAP_FLAGS_MASK = 0xFF; // Only lower 8 bits are map flags
+    constexpr uint32_t MAP_FLAGS_MASK = 0x3F; // Bits 0-5 are map flags
```

Clang (14.0.6)

[warning] variable 'MAP_FLAGS_MASK' is not initialized

(cppcoreguidelines-init-variables)